### ISSUE TITLE
[Bug]: styleElementAttributes doesn't apply to fui-FluentProvider style element

### ISSUE THREAD

**Issue by marcopelegrini** [microsoft/fluentui#25421]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: macOS 13.0
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
    Memory: 132.45 MB / 32.00 GB
    Shell: 5.8.1 - /bin/zsh
  Browsers:
    Chrome: 106.0.5249.119
    Firefox Developer Edition: 107.0
    Safari: 16.1
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://codesandbox.io/s/async-lake-xqhqo5

## Actual Behavior
styleElementAttributes are not applied to fui-FluentProvider1 when using Griffel RendererProvider https://griffel.js.org/react/api/create-dom-renderer 

"random" is added to all styles but fui-FluentProvider1

```
<style nonce="random" data-make-styles-bucket="d"></style>
<style nonce="random" data-make-styles-bucket="t" media="screen and (prefers-reduced-motion: reduce)"></style>
<style nonce="random" data-make-styles-bucket="m" media="screen and (prefers-reduced-motion: reduce)"></style>
<style id="fui-FluentProvider1"></style>
```

## Expected Behavior
`nonce="random"` is also added to `<style id="fui-FluentProvider1"></style>`

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by 1hw7** [comment]
Hi! we are also having this issue. Wondering when there may be plans to integrate nonce support for FluentProvider?

**Comment by layershifter** [comment]
> Hi! we are also having this issue. Wondering when there may be plans to integrate nonce support for FluentProvider?

@1hw7 it's in our current sprint, I will work on it soon 😉

**Comment by 1hw7** [comment]
Awesome, thank you so much @layershifter!!

**Comment by 1hw7** [comment]
If i could make one other suggestion/request (a bit less associated with this bug), it would be awesome to have documentation around how to set the nonce for fluent using griffel. I see documentation in the fluent repo about nonces with fabric ui config and using the cspsettings, but adding reference to the griffel docs @marcopelegrini included would be awesome! (perhaps i'm missing it and it's there, but was not easily able to find docs)

**Comment by layershifter** [comment]
@marcopelegrini Released in `@fluentui/react-components@9.8.0`:

![image](https://user-images.githubusercontent.com/14183168/210521642-7d0c1ec5-6c54-4ef6-8e47-c6052b064cd7.png)

@1hw7 Added 👍  https://react.fluentui.dev/?path=/docs/concepts-developer-advanced-configuration--page
